docs/Correctly-Using-Regular-Expressions.md

```diff
@@ -129,4 +129,10 @@ In addition, ensure your regex is not vulnerable to a Regular Expression Denial
 
 For detailed rationale, along with other information such as contributor credits, see [Correctly Using Regular Expressions for Secure Input Validation - Rationale](./Correctly-Using-Regular-Expressions-Rationale).
 
+## Contributions and Corrections Welcome
+
+If you have any additions, changes, or corrections you'd like to suggest, please [open an issue](https://github.com/ossf/wg-best-practices-os-developers/issues). We appreciate your contributions!
```

Comment:
I like the basic idea! However, instead of *only* recommending issues, we should recommend creating issues *or* pull requests; we shouldn't require unnecessary steps. If there's a simple typo, I'd want them to just create a PR. E.g., something like this:

> If you have any additions, changes, or corrections you'd like to suggest, please [open an issue](https://github.com/ossf/wg-best-practices-os-developers/issues) or [create a pull request](https://github.com/ossf/wg-best-practices-os-developers/pulls). We appreciate your contributions!

Reply:
I see it's been changed! Thanks!